### PR TITLE
Logger

### PR DESCRIPTION
**Pull request by adrnchk**

### CODE REVIEW

File: DogeNet.BLL/Extentions/FluentValidationExtentions.cs [hunk 1]
```diff
@@ -13,5 +13,11 @@ public static class FluentValidationExtentions
             var type = typeof(T);
             return rule.WithMessage("{type} entity is not found");
         }
+
+        public static IRuleBuilderOptions<T, TProperty> ImgUrlValid<T, TProperty>(this IRuleBuilderOptions<T, TProperty> rule)
```
Comment (oleh-leobit) [inline DogeNet.BLL/Extentions/FluentValidationExtentions.cs]: This update should be implemented in scope of other PR?

File: DogeNet.WebApi/Program.cs [hunk 2]
```diff
@@ -4,21 +4,43 @@
...
     using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+    using NLog.Web;
 
     public class Program
     {
         public static void Main(string[] args)
         {
-            CreateHostBuilder(args).Build().Run();
+            var logger = NLogBuilder.ConfigureNLog("NLog.config").GetCurrentClassLogger();
```
Comment (oleh-leobit) [inline DogeNet.WebApi/Program.cs]: Do we really need to make this here?

File: DogeNet.BLL/Extentions/FluentValidationExtentions.cs (hunk 1 above)
Comment (adrnchk) [reply DogeNet.BLL/Extentions/FluentValidationExtentions.cs]: Yes, I will delete this from this pull request (it got 0 references I think I forgot to check out branch some time ago)

File: DogeNet.WebApi/Program.cs (hunk 2 above)
Comment (adrnchk) [reply DogeNet.WebApi/Program.cs]: It can work without it.

**Pull request merged**